### PR TITLE
Editor: Avoid `SetSceneCommnd` when importing file

### PR DESCRIPTION
**Pull request by ycw**
The issue:  Importing 'object' scene(.json) currently performs a mix of 'appending object to default scene'  and 'replacing default scene userData', this is problematic because objects in default scene may have scripts consuming userData of default scene, after importing one/multiple scenes(.json), the default scene's userData get replaced by the last imported one, e.g.:

A simple scene with a plane which has a script consuming `scene.userData`

```
Camera
Scene  <<< holding original scene userData
   Plane 
```  

After importing scene1.json

```
Camera
Scene  <<< holding scene1 userData 
   Plane
   Box   <<< object from scene1
```

After importing scene2.json

```
Camera
Scene  <<< holding scene2 userData
   Plane
   Box   <<< object from scene1
   Ring  <<< object from scene2
```

I.e. It is possible that **when users PLAY the project, it'll throw** because the original scene userData is gone.

----

This PR replaced `SetSceneCommand` by `AddSceneCommand`, so each imported file is imported as a child of default scene, the graph for the above mentioned example would become:

```
Camera
Scene  <<< holding original scene userData
  Plane
- Scene1  <<< holding scene1 userData
    Box 
- Scene2  <<< holding scene2 userData
    Ring
```

This way, the script of Plane can even **access the imported scenes' userData via scene graph traversal**

---
If replacing scene is desired, then I suggest implement `File>Open Project` instead because it has better semantic. 

Implemented, see #28351

### CODE REVIEW

File: editor/js/Loader.js [hunk 1]
```diff
@@ -828,15 +827,7 @@ function Loader( editor ) {
 
 				loader.parse( data, function ( result ) {
 
-					if ( result.isScene ) {
-
-						editor.execute( new SetSceneCommand( editor, result ) );
```
Comment (Mugen87) [inline editor/js/Loader.js:833]: The change makes sense in context of `VRMLLoader`. 

But I'm unsure about `ObjectLoader`. You can export the entire scene to JSON. But when importing it back to the editor, you end up with a different setup (the root scene + the imported scene). I'm not sure this is wanted.

Comment (Mugen87) [reply editor/js/Loader.js:833]: TBH, I'm not sure it's a problem when scene user data are overwritten. I would rather accept that than ending up with a different scene hierarchy after the import.

Right now, it feels like the PR solves one issue by introducing another one.

Comment (ycw) [reply editor/js/Loader.js:833]: The purpose of File>Import is for importing assets to the default scene, not replacing the default scene of any kind.

The more proper action for replacing the default scene,... is File>Open Project (not implemented)

Comment (ycw) [reply editor/js/Loader.js:833]: I edited the original post to clarify the situation :D

Comment (Mugen87) [reply editor/js/Loader.js:833]: #28351 looks good! If @mrdoob likes the save/open pattern, I'm okay with approving this PR as well.

**Comment by Mugen87** [comment]
Do you mind rebasing the PR?

**Comment by Mugen87** [comment]
> Do you mind rebasing the PR?

Never mind! I did it now.

**Comment by ycw** [comment]
thanks

**Pull request merged**